elemica-suggest.coffee

```diff
@@ -108,11 +108,13 @@ elemicaSuggest 0.8.2-SNAPSHOT
               .trigger("element-selected")
             .end()
           .end()
-          .trigger("blur")
           .trigger("focus")
 
         selectionIndicatorTarget( $(element) ).addClass("has-selection")
 
+      currentHighlightedValue = (element) ->
+        $(element).parent().find(".suggestions > .active").text()
```

Comment:
Let's name this `currentlyHighlightedDisplayText`. Per our documentation the `value` is what would be sent to the server. The `display` is what the user is seeing (and typing). They can differ.